src/Motor.Extensions.Diagnostics.Sentry/DefaultHostBuilderExtensions.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Motor.Extensions.Utilities.Abstractions;
+using Sentry;
+using Sentry.Extensions.Logging;
+using MSOptions = Microsoft.Extensions.Options.Options;
+
+namespace Motor.Extensions.Diagnostics.Sentry;
+
+public static class DefaultHostBuilderExtensions
+{
+    public static IMotorHostBuilder ConfigureSentry(this IMotorHostBuilder hostBuilder)
+    {
+        return hostBuilder
+            .ConfigureServices((context, services) =>
+            {
+                var sentryOptions = new SentryLoggingOptions();
+                context.Configuration.GetSection("Sentry").Bind(sentryOptions);
+                if (string.IsNullOrWhiteSpace(sentryOptions.Dsn))
```

Comment:
It seems like this is not even necessary: https://github.com/getsentry/sentry-dotnet/blob/main/src/Sentry/SentrySdk.cs#L72